Octos - Erika Hughes - Solar System

```diff
@@ -0,0 +1,78 @@
+# Create a Solar System class
+class SolarSystem
+
+  attr_accessor :planets
+# Initialize method..
+  def initialize(planets)
+    @planets = planets
+  end
+# add planets..
+  def add_planets(planet)
+    @planets << planet
+  end
+# return list of planets...
+  def list_planets
+    @planets.each do |planet|
+      planet
```

Comment:
This method is never used! You should probably call it in your UI code below to list out all the planets.

I also suspect that this method will not do what you expect. The idea of this method is to return a string, but though the core structure of the loop looks good, I don't see any string work going on.

Making this method work will look very similar to the work we were doing last with with averages and sums:

1. Set up a variable to work with
1. Iterate through the array and modify the variable each time
1. Return the variable at the end

An implementation might look like this:

```ruby
def list_planets
  # Set up a variable to work with
  planet_names = ""
  index = 1

  # Loop through the list and add something to our variable each time
  @planets.each do |planet|
    planet_names += "#{index}. #{planet.name}\n"
    index += 1
  end

  # Return the filled-in variable
  return planet_names
end
```